### PR TITLE
Fix GoTV Ascii Errors

### LINKED ISSUE #95: [BUG] Score stops updating if unicode in steam name
Score stops updating if unicode in steam name.

It has to do with get5_eventapi when it happens the bot has an error.

```
Traceback (most recent call last):
  File "C:\Users\penalte\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\aiohttp\web_protocol.py", line 418, in start
    resp = await task
  File "C:\Users\penalte\Desktop\5vs5\discord-10man-master\utils\server.py", line 109, in _handler
    gotv = server.get_gotv()
  File "C:\Users\penalte\Desktop\5vs5\discord-10man-master\utils\csgo_server.py", line 50, in get_gotv
    status: str = valve.rcon.execute((self.server_address, self.server_port), self.RCON_password, 'status')
  File "C:\Users\penalte\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\valve\rcon.py", line 631, in execute
    return rcon(command)
  File "C:\Users\penalte\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\valve\rcon.py", line 346, in __call__
    raise RCONMessageError("Couldn't decode response: {}".format(exc))
valve.rcon.RCONMessageError: Couldn't decode response: 'ascii' codec can't decode byte 0xe2 in position 602: ordinal not in range(128)
````

### PR DESCRIPTION
**Pull request by yannickgloster**
Closes #95 

Changed method of getting GoTV, also forces all names to be standarized.

### CODE REVIEW

File: cogs/csgo.py [hunk 1]
```diff
@@ -258,19 +257,19 @@ async def pug(self, ctx: commands.Context, *args):
             await player.move_to(channel=team1_channel, reason=f'You are on {team1_captain}\'s Team')
             data = await db.fetch_one('SELECT steam_id FROM users WHERE discord_id = :player',
                                       {"player": str(player.id)})
-            team1_steamIDs.append(data[0])
+            team1_steamIDs[data[0]] = unidecode(player.nick)
```
Comment (lexesj) [inline cogs/csgo.py:260]: Is the value of dict going to be used somewhere in future?

Comment (yannickgloster) [reply cogs/csgo.py:260]: Yep, the steam ID authenticates the user when connecting to the server and then the unidecode(player.nick) is the name that will be forced on the server. Similar to faceit.

Comment (yannickgloster) [reply cogs/csgo.py:260]: https://github.com/splewis/get5#match-schema

**Review by lexesj — APPROVED** [review]
LGTM!

**Pull request merged**